adapters/copy.go

```diff
@@ -9,11 +9,17 @@ import (
 // each obj value refers to where to copy the value to inside `data`
 type Copy struct {
 	CopyPath []string `json:"copyPath"`
+	Path     JSONPath `json:"path"`
```

Comment:
Aren't these two fields functionally identical? I would propose making both of them `JSONPath`:
```golang
type JSONPath []string
```

But then we have to ask, why do we have both in the first place. Is this to support both keys `path` and `copyPath`?

Reply:
yea that was in the story. We can't have multiple json tags for the same field either.

Reply:
Hey there @whatsadebugger, let's just have `CopyPath` support dot delimited string, as opposed to adding the new field `Path`. Conversation regarding this decision can be found here https://www.pivotaltracker.com/story/show/160352011